validator/src/main.rs

```diff
@@ -2512,6 +2543,27 @@ pub fn main() {
         validator_config.max_ledger_shreds = Some(limit_ledger_size);
     }
 
+    validator_config.shred_storage_type = match matches.value_of("rocksdb_shred_compaction") {
+        None => ShredStorageType::default(),
+        Some(shred_compaction_string) => match shred_compaction_string {
+            "level" => ShredStorageType::RocksLevel,
+            "fifo" => {
+                let shred_storage_size = match matches.value_of("rocksdb_fifo_shred_storage_size") {
+                    Some(_) => value_t_or_exit!(matches, "rocksdb_fifo_shred_storage_size", u64),
+                    None => DEFAULT_ROCKS_FIFO_SHRED_STORAGE_SIZE_BYTES,
+                };
```

Comment:
The additional error handling here is unnecessary as clap.rs already takes care of it
```suggestion
                let shred_storage_size = value_t_or_exit!(matches, "rocksdb_fifo_shred_storage_size", u64);
```